src/components/customLoader/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+import colors from '@constants/colors';
```

Comment:
color should be placed in `theme`

Reply:
I will refactor this when I implement the `theme` feature